### DISCUSSION TITLE
A way to reset Filters

### DISCUSSION THREAD

**Discussion by themudassarhassan** [joemasilotti/railsdevs.com#343] [Ideas]
Hey @joemasilotti 
As we are adding more and more filters so wouldn't it be great if there's a way for the folks to reset the filters after applying multiple of them.

**Comment by joemasilotti** [comment]
Agreed! 

UX-wise, we could add a button next to Apply on the mobile filters page. But I'm not sure where we would add one on desktop.

Any ideas?

**Reply by themudassarhassan** [reply]
I'm not sure but one option can be alongside filters.

**Reply by joemasilotti** [reply]
Hmm. So a reset button next to each apply? That might make it seem like you are only resetting that particular filter. 

Maybe it could go next to the search bar when that goes live.

**Reply by themudassarhassan** [reply]
What I meant was to show a button with the text `Reset Filters` right where the `sort` filter is now in the following image
<img width="1147" alt="Screenshot 2022-03-22 at 7 07 37 PM" src="https://user-images.githubusercontent.com/31462903/159500952-506fd9a3-7569-4382-b814-261cff29100c.png">
All the filters would move to left side giving `Reset Filters` button space.

If we show that button right next to search bar wouldn't it imply that it is reseting only search field?

**Reply by joemasilotti** [reply]
Ah! Great idea and good point. Yeah, I think something like that could work.

Maybe even moving the filters/sort closer to the search and pushing the reset button all the way to the right.

**Reply by themudassarhassan** [reply]
Yeah definitely! That would be great.

**Reply by joemasilotti** [reply]
Cool – do you want to run a PR when you finish #341?

**Reply by themudassarhassan** [reply]
Yeah, I would be happy to do that.

**Comment by joemasilotti** [comment]
There is now a Reset button in the form and in the "X of Y developers" banner at the top!
